server/.env.example

```diff
@@ -14,6 +14,7 @@ SIGN_IN_PREFILLED=true
 # DEBUG_MODE=true
 # ACCESS_TOKEN_EXPIRES_IN=30m
 # LOGIN_TOKEN_EXPIRES_IN=15m
+# API_TOKEN_EXPIRES_IN=3650d
```

Comment:
@charlesBochet 10 ans c'est peut-être un peu beaucoup non ? 😅
On peut se dire 1 an pour le moment, mais il faudra un système pour prévenir par email ou autre qu'elle va expirer et qu'il faut en créer une nouvelle

Reply:
1y for no expiry token might be to small. Waiting for @charlesBochet response

Reply:
@charlesBochet Actually I don't think we don't want expiry on the token, usually it happen on many API, but they send you an email some months before so you can create a new one.
It can be 2 years, but in my opinion 10 years is too much

Reply:
ok for 2 years, seen with Charles